src/rfac/crfrs.c

```
@@ -0,0 +1,167 @@
+      denom = R_sqrt(i2 + 4.0 * vi2 * d2 + (y2 * y2) * (vi2 * vi2) * d2_2);
+      if (denom > 0.0) {
+        return deriv / denom;
+      }
+    }
+  }
+
+  denom = R_sqrt(i2 + 4.0 * vi2 * d2);
+  return denom > 0.0 ? (deriv / denom) : 0.0;
+}
+
+static void cr_rs_derivatives(const real *eng, const real *intens, int n,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'cr_rs_derivatives' of similar type ('const int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)